src/main/java/ComputerScienceDepartment.java

```diff
@@ -0,0 +1,4 @@
+
+public class ComputerScienceDepartment extends Department{
+
```

Comment:
I think, putting ArrayList<Enrollment> listOfEnrollemts makes more sense, since the department should know about the enrollments (meaning a way to register students in a certain course)